client/src/components/head-to-head/head-to-head.html

```diff
@@ -0,0 +1,33 @@
+<div class="head-to-head" data-bind="with: fixture()">
+    <div class="title title--wide" data-bind="text: homeTeamName + ' - ' + awayTeamName"></div>
+    <div class="head-to-head__info">
+        <div data-bind="text: new Date(date).toDateString()"></div>
```

Comment:
You have plenty of places like this `new Date(date).toDateString()`. Consider creating a helper that would incapsulate date manipulations.